package.json

```diff
@@ -12,12 +12,12 @@
   },
   "scripts": {
     "start": "node index",
-    "build": "npm run build:grunt && npm run build:handlebars",
+    "build": "(npm run build:grunt || true) && (npm run build:handlebars || true)",
```

Comment:
I get the need for `test`, but I think we should let `build` fail if any of these fail. I can't think of a use case for this.